chain/docker-config/genesis.json

```diff
@@ -69,7 +69,24 @@
         "inflation_min": "0.120000000000000000",
         "goal_bonded": "0.670000000000000000",
         "blocks_per_year": "10519200",
-        "mint_air": false
+        "mint_air": false,
+        "mint_max": [
```

Comment:
what is mint max? Is it max amount of tokens to be retrieved? if so, better use "max_withdrawal_per_time" or something similar, because it is not about minting